internal/character/natasha/ult.go

```diff
@@ -0,0 +1,22 @@
+package natasha
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+func (c *char) Ult(target key.TargetID, state info.ActionState) {
```

Comment:
Ult should do the following in order:
1. Apply E2 to any valid characters
2. Perform party-wide heal
3. Add 5 energy to natasha